src/auth0-sso-login.js

```diff
@@ -174,9 +171,19 @@ export default class auth {
 
     this.errorHandler.tryCaptureError();
 
-    let completedRedirect = this.redirectHandler.attemptRedirect();
-    if (completedRedirect) {
-      return Promise.resolve();
+    let redirectFromAuth0Result = await new Promise((resolve, reject) => this.auth0ClientProvider.getClient().parseHash({}, (error, authResult) => error ? reject(error) : resolve(authResult)));
+    let containsToken = redirectFromAuth0Result && redirectFromAuth0Result.idToken && redirectFromAuth0Result.accessToken
+    if (containsToken) {
```

Comment:
Only to reconfirm. On second login (when there is already a valid session) the response from auth0 does not contain token, so it will skip this if block an continue with `renewAuth` bellow, correct?

Reply:
correct.